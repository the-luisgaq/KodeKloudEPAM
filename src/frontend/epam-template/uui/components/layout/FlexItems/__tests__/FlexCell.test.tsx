import React from 'react';
import { FlexCell } from '../FlexCell';
import { render } from '@epam/uui-test-utils';

describe('FlexCell', () => {
    it('should be rendered correctly', () => {
        const { asFragment } = render(<FlexCell />);
        expect(asFragment()).toMatchSnapshot();
    });

    it('should be rendered correctly with props', () => {
        const { asFragment } = render(<FlexCell onClick={ () => {} } width="100%" minWidth={ 120 } alignSelf="center" grow={ 1 } shrink={ 1 } textAlign="left" />);
        expect(asFragment()).toMatchSnapshot();
    });
});
