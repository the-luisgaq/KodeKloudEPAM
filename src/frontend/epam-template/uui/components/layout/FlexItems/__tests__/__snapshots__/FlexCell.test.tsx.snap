// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlexCell should be rendered correctly 1`] = `
<DocumentFragment>
  <div
    class="flexCell"
    style="min-width: 0; flex-basis: 0px;"
  />
</DocumentFragment>
`;

exports[`FlexCell should be rendered correctly with props 1`] = `
<DocumentFragment>
  <div
    class="flexCell"
    style="min-width: 120px; flex-grow: 1; flex-shrink: 1; flex-basis: 100%; text-align: left; align-self: center;"
  />
</DocumentFragment>
`;
